include/posix/time.h

```diff
@@ -10,7 +10,25 @@
 extern "C" {
 #endif
 
+#ifdef CONFIG_NEWLIB_LIBC
+/* Kludge to support outdated newlib version as used in SDK 0.10 for Xtensa */
+#include <newlib.h>
+
+#ifdef __NEWLIB__
```

Comment:
Can you add a comment of which case is handling the old xtensa version

Reply:
> Can you add a comment of which case is handling the old xtensa version

Done.